jscomp/core/matching_polyfill.ml

```diff
@@ -23,6 +23,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
 let () = Ast_untagged_variants.extract_concrete_typedecl := Ctype.extract_concrete_typedecl
+let () = Runtime_representation.extract_concrete_typedecl := Ctype.extract_concrete_typedecl
```

Comment:
Any idea why this file is called "polyfill"?
The only reason I can think about for its existence is: hack to avoid recursion amongst modules. If that's not the role, then perhaps this file should not exist.

Reply:
No idea. IIRC this is a "random" file I put the assignment in just to get it there.